docs/src/components/prototyper/prototyper.jsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+
+import ExecutionEnvironment from '@docusaurus/ExecutionEnvironment';
+import ThemeContext from '@theme/theme-context';
+import { isProductionBuild, uuidv4 } from '@site/src/utils';
+import PrototyperContext from './context';
+
+const jsonHeaders = {
+  'Accept': 'application/json',
+  'Content-Type': 'application/json'
+};
+const trackerPollingInterval = 2000;
+
+
+const Prototyper = ({children, startPrototyperApi, trainModelApi, chatBlockSelector, chatBlockScriptUrl}) => {
+    const [trackingId, setTrackingId] = React.useState(null);
+    const [hasStarted, setHasStarted] = React.useState(false);
+    const [projectDownloadUrl, setProjectDownloadUrl] = React.useState(null);
+    const [trainingData, setTrainingData] = React.useState({});
+    const [pollingIntervalId, setPollingIntervalId] = React.useState(null);
+    const [hasTrained, setHasTrained] = React.useState(false);
+    const [isTraining, setIsTraining] = React.useState(false);
+
+    // FIXME: once we can use `rasa-ui` outside of `rasa-x`, we can remove this
```

Comment:
Really not proud of this, but couldn't find a better way to do this. The `<script>` tags are not read in the MDX files 🤷 . It'll be cleaned up anyway when we make `rasa-ui` available outside of `rasa-x`